### PR TITLE
Fix AIX dlopen debug log.

### PR DESCRIPTION
**Pull request by likema**

### CODE REVIEW

File: ACE/ace/DLL_Manager.cpp [hunk 1]
```diff
@@ -203,20 +203,6 @@ ACE_DLL_Handle::open (const ACE_TCHAR *dll_name,
...
 
-                  if (ACE::debug ())
-                    {
-                      ACE_TString err;
-                      ACELIB_DEBUG ((LM_DEBUG,
-                                  ACE_TEXT ("ACE (%P|%t) DLL_Handle::open ")
-                                  ACE_TEXT ("(\"%s\", 0x%x) -> %s: %s\n"),
-                                  aix_pathname,
-                                  open_mode,
-                                  (this->handle_ != ACE_SHLIB_INVALID_HANDLE
```
Comment (likema) [inline ACE/ace/DLL_Manager.cpp]: handle_ is always ACE_SHLIB_INVALID_HANDLE

Comment (jwillemsen) [reply ACE/ace/DLL_Manager.cpp]: This probably should move below the ACE_OS::dlopen which was on line 220, it prints the result of the dlopen

Comment (likema) [reply ACE/ace/DLL_Manager.cpp]: Yes, your are right.

Comment (likema) [reply ACE/ace/DLL_Manager.cpp]: Fixed

**Pull request merged**